### PR TITLE
Responsive creation of post

### PR DESCRIPTION
**Pull request by tiagolimpe**
**Feature/Bug description:** The creation of post is not responsive. PDF files with long title overflow the container and the thumbnails of pictures are in the left side.
screenshot_2018-10-08_18-34-13
screenshot_2018-10-08_18-34-29
screenshot_2018-10-08_18-34-37
screenshot_2018-10-08_18-34-50

**Solution:** Making responsive. Fix the PDF file with long titles and centralizing the thumbnails of pictures. Also, improving the survey creation in small screens.

screenshot_2018-10-08_18-36-13
screenshot_2018-10-08_18-36-22
screenshot_2018-10-10_17-36-25
screenshot_2018-10-10_17-19-51

**TODO/FIXME:** n/a

### CODE REVIEW

File: frontend/post/save_post.html [hunk 1]
```diff
@@ -1,120 +1,124 @@
...
+                  <div ng-messages='saveForm.message.$error'>
+                      <div ng-message="required">O post deve conter um texto!</div>
                   </div>
-                  <div ng-if="postCtrl.typePost === 'Survey'">
-                    <md-icon>edit</md-icon>
-                    Alternar para postagem comum
+                </md-input-container>
+                <md-input-container ng-show="postCtrl.showVideoUrlField()" class="md-block" style="margin-bottom: 0;">
+                  <label>Link do Video</label>
+                  <textarea style="max-height: 400px;overflow-y:auto;resize:none;" ng-model="postCtrl.post.video_url"
```
Comment (JuliePessoa) [inline frontend/post/save_post.html]: move style to css

File: frontend/post/postDirective.js [hunk 2]
```diff
@@ -367,7 +367,8 @@
         };
 
         postCtrl.showPlaceholderMsg = function() {
-            return postCtrl.isTyping() ? "Título" : "Escreva aqui uma nova publicação";
+            return postCtrl.isTyping() ? "Título" : !postCtrl.isTyping() && screen.width < 959 ?
```
Comment (raonismaneoto) [inline frontend/post/postDirective.js]: I think you could use only the first option

File: frontend/post/save_post.html [hunk 3]
```diff
@@ -1,120 +1,124 @@
...
-        <img ng-src="{{ postCtrl.getInstPhotoUrl()}}"
-        title="{{postCtrl.user.current_institution.name}}"
-        style="border-radius: 50%;" width="50px" height="50px"/>
-      </div>
-      <div flex>
+<md-card class="create-post-container-resp create-post-margin">
+    <form name="saveForm" prevent-state-change="postCtrl.post"
+    ng-submit="postCtrl.save(isEditing, originalPost, posts, saveForm)">
+    <div >
+      <md-card-content layout="row" style="padding:0;" layout-padding>
```
Comment (raonismaneoto) [inline frontend/post/save_post.html]: padding:0 and layout-padding sounds weird, are they really necessary?

File: frontend/post/save_post.html [hunk 4]
```diff
@@ -1,120 +1,124 @@
...
-                  <div ng-message="pattern">Link do Youtube inválido.</div>
-                </div>
-              </md-input-container>
-          </div>
-            <!-- SURVEY -->
-          <survey-directive ng-if="postCtrl.typePost === 'Survey' && !isEditing" post="postCtrl.post" posts="posts" user="postCtrl.user" callback="postCtrl.choiceCommon"
-              options="postCtrl.options"></survey-directive>
+          <img ng-src="{{ postCtrl.getInstPhotoUrl()}}"
+          title="{{postCtrl.user.current_institution.name}}"
+          style="border-radius: 50%;" width="50px" height="50px"/>
```
Comment (raonismaneoto) [inline frontend/post/save_post.html]: It should be in the css' file

File: frontend/post/save_post.html [hunk 5]
```diff
@@ -1,120 +1,124 @@
...
-                  {{!postCtrl.showFiles() ? 'Anexar PDF' : 'Trocar PDF'}}
-                </md-button>
-              </md-menu-item>
-              <md-menu-item ng-if="!isEditing">
-                <md-button ng-click="postCtrl.setTypeOfPost()" aria-label="Tipo do Post">
-                  <div ng-if="postCtrl.typePost === 'Common'">
-                    <md-icon>poll</md-icon>
-                    Criar Enquete
+        <div flex>
+          <div>
```
Comment (raonismaneoto) [inline frontend/post/save_post.html]: is it necessary?

File: frontend/post/save_post.html [hunk 6]
```diff
@@ -1,120 +1,124 @@
...
+            <div ng-if="postCtrl.typePost === 'Common'"  layout="column">
+                <md-input-container md-no-float class="md-block" style="margin: 0px; padding-top: 10px;">
```
Comment (raonismaneoto) [inline frontend/post/save_post.html]: this style should be in the css' file

**Pull request merged**